app/test_split_integer.py

```diff
@@ -2,20 +2,20 @@
 
 
 def test_sum_of_the_parts_should_be_equal_to_value() -> None:
-    pass
+    assert sum(split_integer(52, 6)) == 52, "Sum of parts should be equal to value"
 
 
 def test_should_split_into_equal_parts_when_value_divisible_by_parts() -> None:
-    pass
+    assert (split_integer(8, 2) == [4, 4]), f"Array should split value 8 on two equal parts not {split_integer(8, 2)}"
```

Comment:
You are calling `split_integer(8, 2)` twice in this assertion. Although it's not against the checklist, it's generally more efficient to call the function once and store the result in a variable if you need to use it multiple times.